test/test_jit_fuser.py

```diff
@@ -60,6 +62,21 @@ def func(x):
     def test_abs_cpu(self):
         self._test_fused_abs()
 
+    @unittest.skipIf(IS_SANDCASTLE, "NYI: fuser CPU support for Sandcastle")
+    @enable_cpu_fuser
```

Comment:
Nit: should we mark this test as Windows only, as it is a bit useless for other platform that use the hardcoded tmp path `/tmp`?

Reply:
That's a valid point. On Windows the login user name may very likely contain Unicode characters, and the default temp path is `C:\Users\USERNAME\AppData\Local\Temp`. On other platforms I cannot think of a way it will contains Unicode characters, either.

Reply:
@skyline75489 That's not the point. Since the jit fuser uses `/tmp/` as the temporary directory (see https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/codegen/fuser/cpu/fused_kernel.cpp#L41-L42), there is no way it will get a Unicode-related problem on platforms except Windows. So would you please skip this case for those platforms?

Reply:
Done.